docs/README.md

```diff
@@ -1,154 +1,233 @@
 # Docker Documentation
 
-The source for Docker documentation is here under `sources/` and uses extended
-Markdown, as implemented by [MkDocs](http://mkdocs.org).
+The source for Docker documentation is in this directory under `sources/`. Our
+documentation uses extended Markdown, as implemented by
+[MkDocs](http://mkdocs.org).
 
-The HTML files are built and hosted on
-[http://docs.docker.com](http://docs.docker.com), and update automatically
-after each change to the `docs` branch of [Docker on
-GitHub](https://github.com/docker/docker) thanks to post-commit hooks.
+Through the use of post-commit hooks, Docker builds and updates our static HTML documentation automatically after each change to the `docs` branch of [this repository](https://github.com/docker/docker). You can find the resulting documentaiton hosted on [http://docs.docker.com](http://docs.docker.com).
```

Comment:
Need to wrap lines at 80 chars. 

Typo: documentaiton

Please make link text descriptive (as opposed to "click here" or "this repo").